[WFLY-16958]: Operations may fail on JMS Topic.

ehsavoie opened the pull request:
* If the topic already has the 'jms.topic.' it might not be found when executing runtime operations.

Jira: https://issues.redhat.com/browse/WFLY-16958

Signed-off-by: Emmanuel Hugonnet <ehugonne@redhat.com>

bstansberry commented on messaging-activemq/subsystem/src/main/java/org/wildfly/extension/messaging/activemq/jms/JMSTopicControlHandler.java:
```diff
@@ -143,11 +143,14 @@ protected void executeRuntimeStep(OperationContext context, ModelNode operation)
 
         final String operationName = operation.require(ModelDescriptionConstants.OP).asString();
-        final String topicName = PathAddress.pathAddress(operation.require(ModelDescriptionConstants.OP_ADDR)).getLastElement().getValue();
+        String topicName = PathAddress.pathAddress(operation.require(ModelDescriptionConstants.OP_ADDR)).getLastElement().getValue();
```
Please use OperationContext.getCurrentAddressValue();

ehsavoie replied:
Of course :)

ehsavoie committed:
[WFLY-16958]: Operations may fail on JMS Topic.

* If the topic already has the 'jms.topic.' it might not be found when
  executing runtime operations.

Jira: https://issues.redhat.com/browse/WFLY-16958

Signed-off-by: Emmanuel Hugonnet <ehugonne@redhat.com>

bstansberry commented:
Thanks @ehsavoie
